src/main/resources/application.conf.example

```diff
@@ -17,5 +17,6 @@ hubot {
 hipchat {
   jid="111@chat.hipchat.com"
   password="somePass"
-  chatAlias = "ScalaBot" //cant be same as name, need to read XEP to see why 
+  chatAlias = "ScalaBot" //cant be same as name, need to read XEP to see why
+  rooms = ["somenumber_somename@conf.hipchat.com"] //whitelist of rooms, defaults to every room if not set
```

Comment:
@mattfielder  we should probably leave a slightly more detailed comment here that this needs to be set to 
rooms = [] 
to ensure every room is joined automatically